content/SCALE/SCALECLIReference/Auth/CLISessions.md

```diff
@@ -12,4 +12,225 @@ tags:
 
 {{< include file="/_includes/CLIGuideWIP.md" type="page" >}}
 
+## Sessions Commands
+
+The **sessions** namespace has seven commands and is based on functions found in the SCALE API and web UI. 
+It provides access to session information through the seven **sessions** commands. 
+
+You can enter commands from the main CLI prompt or the **sessions** namespace prompt.
+
+### Sessions Command
+
+The `sessions` command returns a table that displays each session id, whether or not each session is current and internal, each session origin, and the credentials and credential data used to log into each session.
+
+{{< expand "Displaying all sessions since system boot" "v" >}}
+#### Description
+The `sessions` command does not require entering property or arguments.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table with the ID, current session state, origin such as an IP or socket information, credentials, and creation date and time.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table with the ID, current session state, origin such as an IP or socket information, credentials, and creation date and time.
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions`
```

Comment:
I think we are standardizing on the full namespace path as the example. This is not incorrect but we should use the same string start point.
```suggestion
`auth sessions`
```